main.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+
+const int SIZE = 20;
+int unsorted[SIZE] = {3,5,7,2,1,9,4,6,8,0,19,16,12,10,18,11,17,14,13,15};
+
+int main(int argc, char *argv[])
+{
+  std::string datafile = "";
+  if (argv[1] != nullptr)
+   {
+   std::string arg = static_cast<std::string>(argv[1]);
+
+   }
+    std::string arg2 = static_cast<std::string>(argv[2]);
+
+
+ if (arg2 = "sort" )    {
+
+    std::cout << "Please use ascending (asc) or descending (desc)" << std::endl;
+    //Should I use a 3rd argument here or what? ^
```

Comment:
Yes, this is where argument 3 comes in. You would also use it for the search function to pass the string to search for.